src/Raven.Server/RavenServer.cs

```diff
@@ -3080,9 +3079,9 @@ private static async Task<License> GetLicenseFromApi(License license, Transactio
         {
             try
             {
-                var response = await LicenseManager.GetUpdatedLicenseResponseMessage(license, contextPool)
+                var response = await LicenseManager.GetUpdatedLicenseResponseMessage(license, contextPool, CancellationToken.None)
```

Comment:
why `CancellationToken.None`?

Reply:
Because we're in a static method here. In other places where `GetUpdatedLicenseResponseMessage()` is called we pass the shutdown token of the server store.